src/v2/api/user/dto/CreateContactDTO.ts

```diff
@@ -7,6 +7,11 @@ export class CreateContactDTO {
     @IsNotEmpty({
       message: 'name can not be empty',
     })
+    @Matches(
+      /^[a-zA-Z0-9AБВГДЕЄЖЗИІЇЙКЛМНОПРСТУФХЦЧШЩЬЮЯабвгдеєжзиіїйклмнопрстуфхцчшщьюя\- ]+$/,
```

Comment:
the name of contact can consist of not only ukrainian letters, but also english letters and numbers